### DISCUSSION TITLE
Multiple colors

### DISCUSSION THREAD

**Discussion by drpoposky** [fabiocaccamo/django-colorfield#92] [Ideas]
It would be interesting to have a field that let's you combine more than 1 colour.
Just like football teams (red-black for milan or black-lightblue for Inter in Italy A league).

So this field you can chose how many colours to add just pulling next to each other.
This could be still a CharField with the possibility to replicate the pattern n times and see from the admin the colours just next to each other.

**Comment by fabiocaccamo** [comment]
I think that the amount of work needed for supporting this would be not rewarded by the advantages obtained, considering also that the same result could be easily achieved by just adding two color fields to the model.

Is there any benefit that I am not considering about this proposal?

**Comment by drpoposky** [comment]
Yes,
I imagine this is pretty complicated for small benefits in small occasions.
Yes, I thought that one could just add 2 columns (just like the football-team case)

But there are some cases (maybe very rare) that you don't know in advance how many colors you need.
Political party case:
each party could be associated with a color but when a government is formed we associate the majority/minority coalition with the pairing of these colors:

for example, in Italy we had:
- Governo Conte I  (yellow-green)
- Governo Conte II (yellow-red)
- Governo Draghi (red-yellow-green-blue)

But in the exact moment I am writing you I think this case could be approached just using one color for each party and then with a property get the list of colors associated to gov since each gov is associated with a party.

**Comment by fabiocaccamo** [comment]
I understand better now, thank you for the explanation.
Yes, in some cases there could be the need of having a list of colors of undefined length without using foreign keys, in those cases having a `ColorListField` could be useful.
